h5_test/tst_h_atts3.c

```diff
@@ -148,8 +152,14 @@ main()
       for (i = 0; i < num_obj; i++)
       {
 	 /* Get the name, and make sure this is a type. */
+
+#if H5_VERSION_GE(1,12,0)
+	 if (H5Oget_info_by_idx3(grpid, ".", H5_INDEX_CRT_ORDER, H5_ITER_INC,
+				 i, &obj_info, H5O_INFO_BASIC, H5P_DEFAULT) < 0) ERR;
```

Comment:
Are there any places in the netCDF code that may use more of the information that gets returned here? If H5O_INFO_BASIC is enough, great. Just want to make sure that H5O_INFO_ALL wouldn't be more appropriate.